errors.go

```diff
@@ -16,18 +16,18 @@ var ErrWriter io.Writer = os.Stderr
 
 // MultiError is an error that wraps multiple errors.
 type MultiError struct {
-	Errors []error
+	errors []error
```

Comment:
Is there a reason we don't want to expose the list of errors anymore? I feel like that is useful to the user (e.g. to be able to iterate and format some sort of output if they choose). If we choose not to expose that member, I think it would be reasonable to make the struct itself private.